src/scenes/quote/Quote.jsx

```diff
@@ -0,0 +1,40 @@
+import './Quote.css'
+import backgroundImage from './bg_quote.jpg'
+import Quotes from './quotes.json'
+
+/**
+ * A component that displays an inspirational quote that changes daily.
+ * Quotes are pulled from a list of 367 quotes.
+ * 
+ * @component
+ * @author Ada Vuorinen
+ */
+function Quote(props) {
+
+  // Calculate the day of the year as integer between 1-366
+  // First, get today's date and time
+  let date = new Date()
+  // Convert to UTC to avoid DST problems and subtract start of year
+  let dayOfYear = Math.floor((Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()) - Date.UTC(date.getFullYear(), 0, 0)) / 24 / 60 / 60 / 1000)
```

Comment:
Tämä taitaa laskea päivän, kun komponentti renderöidään. Tässä sovelluksessa voi tulla tilanne, että komponenttia ei renderöidä päivittäin, eikä erityisesti kun vuorokausi vaihtuu. Uudelleenrenderöinti tapahtuu, kun sisältö muuttuu, muuten sovellus toistaa renderöityjä näkymiä loputtomiin.